stafihub/chain.json

```diff
@@ -30,19 +30,19 @@
   },
   "codebase": {
     "git_repo": "https://github.com/stafihub/stafihub",
-    "recommended_version": "v0.2.3",
+    "recommended_version": "v0.4.1",
     "compatible_versions": [
-      "v0.2.3"
+      "v0.4.1"
     ],
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/stafihub/network/main/mainnets/stafihub-1(dragonberry)/genesis.json"
     },
     "versions": [
       {
-        "name": "v0.2.3",
```

Comment:
Can we please keep v0.2.3 in the versions array, and then just add v0.4.1 after it?

Reply:
Sure. Please check again. Thanks!